packages/components/src/DateTimePickers/views/DateTimeView/DateTimeView.scss

```diff
@@ -14,3 +14,9 @@
 		padding-left: $padding-normal;
 	}
 }
+
+.footer {
+	flex-grow: 1;
+	text-align: right;
+	padding: 10px;
```

Comment:
This introduce extra padding on left and right, making the button not aligned to the rest.
And use padding variables please.
`padding: $padding-small 0;`